libethdrivers/include/ethdrivers/zynq7000.h

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright 2017, DornerWorks, Ltd.
+ *
+ * This software may be distributed and modified according to the terms of
+ * the GNU General Public License version 2. Note that NO WARRANTY is provided.
+ * See "LICENSE_GPLv2.txt" for details.
+ *
+ * @TAG(NICTA_GPL)
+ */
+
+#ifndef __ETHIF_ZYNQ7000_DRIVER_H
+#define __ETHIF_ZYNQ7000_DRIVER_H
+
+#include <platsupport/io.h>
+#include <ethdrivers/raw.h>
+
+#define ZYNQ7000_INTERRUPT_ETH0 54
+
+/**
+ * This function initialises the hardware and conforms to the ethif_driver_init
+ * type in raw.h
+ * @param[out] eth_driver   Ethernet driver structure to fill out
+ * @param[in] io_ops        A structure containing os specific data and
+ *                          functions.
+ * @param[in] config        Unused, should be NULL
```

Comment:
this is true right now in this commit, but config becomes a required parameter later without this being updated

Reply:
You're right. We will update the comment.